libs/ledger-live-common/src/ble/hooks/useBleDevicePairing.ts

```diff
@@ -31,10 +30,10 @@ export const useBleDevicePairing = ({
   const [pairingError, setPairingError] = useState<PairingError>(null);
 
   useEffect(() => {
-    const requestObservable = withDevice(deviceId)(t => from(getVersion(t))).pipe(first());
+    const requestObservable = withDevice(deviceId)(t => from(getAppAndVersion(t))).pipe(first());
```

Comment:
It's a great idea ! Though i think you need to update `libs/ledger-live-common/src/ble/hooks/useBleDevicePairing.test.ts` so it mocks `getAppAndVersion` now (and not `getVersion`) ;)

Reply:
Good catch, pushed.